### PR TITLE
refactor: set queries and messages in right order for modules

### PR DESCRIPTION
**Pull request by lubtd**
Many changes we did in the last few weeks changed the order of messages and queries where there are listed

This PR set them in the right order

### CODE REVIEW

**Review by aljo242 — CHANGES_REQUESTED** [review]
Looks good to me - we may as well make some other additions based on some tiny nitpicks

File: proto/monitoringp/query.proto [hunk 1]
```diff
@@ -14,47 +14,54 @@ option go_package = "github.com/tendermint/spn/x/monitoringp/types";
...
-  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
-    option (google.api.http).get = "/tendermint/spn/monitoringp/params";
-  }
   // Queries a ConsumerClientID by index.
   rpc ConsumerClientID(QueryGetConsumerClientIDRequest) returns (QueryGetConsumerClientIDResponse) {
     option (google.api.http).get = "/tendermint/spn/monitoringp/consumer_client_id";
   }
+
   // Queries a ConnectionChannelID by index.
 	rpc ConnectionChannelID(QueryGetConnectionChannelIDRequest) returns (QueryGetConnectionChannelIDResponse) {
```
[the remark this answers is not in the document]
Comment (lubtd) [reply proto/monitoringp/query.proto:23]: Yep, I don't see those in the IDe, I will address all of them

File: proto/reward/query.proto [hunk 2]
```diff
@@ -27,16 +23,12 @@ service Query {
     option (google.api.http).get = "/tendermint/spn/reward/reward_pool";
   }
 
-  // this line is used by starport scaffolding # 2
-}
-
-// QueryParamsRequest is request type for the Query/Params RPC method.
-message QueryParamsRequest {}
+  // Parameters queries the parameters of the module.
```
[the remark this answers is not in the document]
Comment (lubtd) [reply proto/reward/query.proto:26]: We actually don't need this comment

File: proto/types/ibc.proto [hunk 3]
```diff
@@ -7,33 +7,33 @@ option go_package = "github.com/tendermint/spn/pkg/types";
 
 // MerkelRool represents a Merkel Root in ConsensusState
 message MerkelRool {
```
[the remark this answers is not in the document]
Comment (aljo242) [reply proto/types/ibc.proto:9]: Lol yes, my bad!

MerkelRoot

**Review by aljo242 — APPROVED** [review]

**Pull request merged**